node/src/p2p/callbacks/p2p_callbacks_actions.rs

```diff
@@ -46,6 +47,11 @@ pub enum P2pCallbacksAction {
     RpcRespondBestTip {
         peer_id: PeerId,
     },
+    P2pPubsubValidateMessage {
+        message_content: GossipNetMessageV2,
```

Comment:
Just for consideration, because I am still not sure: do `message_content` and `message` need to be part of the action? that is, instead of being part of the state and the actions just having a reference to them (through some id)